governance/program/tests/program_test/mod.rs

```diff
@@ -1,6 +1,9 @@
+use std::borrow::Borrow;
```

Comment:
Is this needed?

Reply:
Yes, to deserialise bitcode based UpgradeableLoaderState account